src/encoded/static/components/cart/cart.js

```diff
@@ -598,7 +613,7 @@ const requestDatasets = (elements, fetch, session) => {
         sessionPromise = fetch('/session');
     } else {
         // We have a session CSRF token, so retrieve it immediately.
-        sessionPromise = Promise.resolve(session._csrft);
```

Comment:
How did **session_csrft** ever work? The backend seems to need have it set as **session_csrft_**